dali/test/python/test_dali_tf_dataset.py

```diff
@@ -58,8 +61,8 @@ def __init__(self, batch_size, num_threads, device_id = 0, seed = 0):
         self.res_uniform = ops.Uniform(range = (256.,480.))
         self.uniform = ops.Uniform(range = (0.0, 1.0))
         self.cast = ops.Cast(
-            device = "cpu",
-            dtype = types.FLOAT16)
+            device = device,
+            dtype = types.INT16)
```

Comment:
We don't support FLOAT16 on the GPU, so I've changed it to INT16 to have the same test both for the CPU and the GPU.

Reply:
I'll fix the Cast operator in a separate PR coming soon